From: Simon McVittie <smcv@debian.org>
Date: Mon, 19 Feb 2024 13:04:14 +0000
Subject: test-admin-deploy-var: Don't rely on OSTREE_FEATURES

This is set during build-time testing, but unset during "as-installed"
tests.

Bug: https://github.com/ostreedev/ostree/issues/3183
Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://github.com/ostreedev/ostree/pull/3184
---
 tests/test-admin-deploy-var.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-admin-deploy-var.sh b/tests/test-admin-deploy-var.sh
index 3079490..fc61576 100755
--- a/tests/test-admin-deploy-var.sh
+++ b/tests/test-admin-deploy-var.sh
@@ -21,7 +21,7 @@ set -euox pipefail
 
 . $(dirname $0)/libtest.sh
 
-if ! echo "$OSTREE_FEATURES" | grep --quiet --no-messages "initial-var"; then
+if ! has_ostree_feature initial-var; then
     fatal missing initial-var
 fi
 
